std/string.d

```diff
@@ -1193,6 +1236,37 @@ ptrdiff_t lastIndexOf(Char1, Char2)(const(Char1)[] s, const(Char2)[] sub,
     return -1;
 }
 
+/// Ditto
+ptrdiff_t lastIndexOf(Char1, Char2)(const(Char1)[] s, const(Char2)[] sub,
+        in size_t startIdx, in CaseSensitive cs = CaseSensitive.yes) @safe pure
```

Comment:
What's this supposed to if both s and sub are empty? Could you please add a unittest for that?